cylc/flow/tui/app.py

```diff
@@ -310,11 +310,10 @@ def get_snapshot(self):
                 }
             )
         except WorkflowStopped:
-            self.client = None
             return dummy_flow({
                 'name': self.reg,
                 'id': self.reg,
-                'status': 'stopped',
+                'status': 'Workflow not found',
```

Comment:
But what if the workflow is genuinely stopped?

Need to check manually before setting "workflow not found" as the status.

Reply:
That was what my question on the issue was getting at